qdev_wrappers/__init__.py

```diff
@@ -2,4 +2,9 @@
 from qdev_wrappers.device_annotator.device_image import save_device_image
 from qdev_wrappers.show_num import show_num
 from qdev_wrappers.sweep_functions import do0d, do1d, do2d, do1dDiagonal
+
 from qcodes.monitor.monitor import Monitor
+from qcodes.instrument.base import Instrument
```

Comment:
I think this is unused now?

Reply:
This might break things if people are not importing the instrument correctly...

Reply:
Ok fine to leave it